test/common/test_io.cpp

```diff
@@ -51,29 +51,6 @@ PointXYZRGBA pt_xyz_rgba, pt_xyz_rgba2;
 PointXYZRGB pt_xyz_rgb;
 PointXYZ pt_xyz;
 
-///////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
Should we keep these tests? @mvieth

Reply:
These tests are duplicated from test/common/test_common.cpp (https://github.com/PointCloudLibrary/pcl/blob/master/test/common/test_common.cpp#L364)

Reply:
Ahh, alright.